bitnami/common/Chart.yaml

```diff
@@ -20,4 +20,4 @@ sources:
   - https://github.com/bitnami/charts
   - https://www.bitnami.com/
 type: library
-version: 1.14.1
+version: 1.14.2
```

Comment:
Could you please also bump the `appVersion` field to this same value? Both versions should match (although they were not matching at this moment)

Reply:
I did it by myself, so we can merge this PR